content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -316,11 +316,11 @@ Currently, storage size is the only resource that can be set or requested.  Futu
 
 ### Volume Mode
 
-{{< feature-state for_k8s_version="v1.13" state="beta" >}}
+{{< feature-state for_k8s_version="v1.18" state="stable" >}}
 
 Prior to Kubernetes 1.9, all volume plugins created a filesystem on the persistent volume.
```

Comment:
I'd drop this line or reword it. The aim here is to treat block and filesystem volumes as equally valid peers.

Reply:
(because both are graduated features, once this lands)

Reply:
Removed "Prior to Kubernetes 1.9" line